Formula/conductr-cli.rb

```diff
@@ -0,0 +1,14 @@
+class ConductrCli < Formula
+  desc "The Lightbend ConductR CLI"
+  homepage "https://conductr.lightbend.com"
+  url "https://bintray.com/lightbend/generic/download_file?file_path=conductr-cli-1.0.1-Mac_OS_X-x86_64.zip"
```

Comment:
It isn't possible to link to a releases based tar ball as we must manually produce a native image via pyinstaller. Happy to receive other thoughts. Thanks.